tests/checkpoint/test_checkpoint.py

```diff
@@ -1573,7 +1625,6 @@ def test_newstyle_checkpoint_instantiates_and_produces_a_validation_result_when_
         data_context=context,
         config_version=1,
         run_name_template="%Y-%M-foo-bar-template",
-        expectation_suite_name="my_expectation_suite",
```

Comment:
By deleting the `expectation_suite_name` argument, the inconsistency between `validator.expectation_suite_name` and the argument is eliminated (otherwise, error message would appear).